src/Modules/Main.lua

```diff
@@ -50,16 +50,23 @@ function main:Init()
 	self.modes["LIST"] = LoadModule("Modules/BuildList")
 	self.modes["BUILD"] = LoadModule("Modules/Build")
 
-	if launch.devMode or (GetScriptPath() == GetRuntimePath() and not launch.installedMode) then
-		-- If running in dev mode or standalone mode, put user data in the script path
-		self.userPath = GetScriptPath().."/"
+	self.popups = { }
+	--if launch.devMode or (GetScriptPath() == GetRuntimePath() and not launch.installedMode) then
+	--	-- If running in dev mode or standalone mode, put user data in the script path
+	--	self.userPath = GetScriptPath().."/"
+	--else
+	local invalidPath
+	self.userPath, invalidPath = GetUserPath()
+	if not self.userPath then
+		self:OpenPathPopup(invalidPath)
 	else
-		self.userPath = GetUserPath().."/Path of Building/"
+		self.userPath = self.userPath.."/Path of Building/"
 		MakeDir(self.userPath)
+		--end
+		self.defaultBuildPath = self.userPath.."Builds/"
+		self.buildPath = self.defaultBuildPath
```

Comment:
Is this path taken if we're going through `OpenPathPopup` in the branch above, in some sort of external loop or something? If not; it will lead to there being no `Path of Building` and `Builds` subdirs, nor will the buildpaths be set.